ex3/InstallJenkins.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+which docker > /dev/null 2>&1
+if [[ $? -eq 1 ]]; then        
+        yum update -y 
+        yum install -y yum-utils device-mapper-persistent-data lvm2
+        yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+        echo "Installing docker....."
+        yum install -y docker-ce docker-ce-cli containerd.io
+        systemctl enable docker
+        systemctl start docker
+
+        systemctl is-active --quiet docker && echo Docker is running
+else
+        echo "docker installed"
+fi
+
+docker container inspect myJenkins > /dev/null 2>&1
+if [[ $? -eq 1 ]]; then                
+        echo "Installing jenkins"
+	mkdir -p /home/shai/Documents/backery/playground/jcasc/ex3/JenkinsHome
```

Comment:
same as above. accept the path as parameter.

Reply:
I changed the /home/shai to ~